graphql/resolvers.go

```diff
@@ -972,6 +972,19 @@ func (r *patchResolver) ID(ctx context.Context, obj *restModel.APIPatch) (string
 	return *obj.Id, nil
 }
 
+func (r *patchResolver) TriggerAliases(ctx context.Context, obj *restModel.APIPatch) ([]*PatchTriggerAlias, error) {
```

Comment:
can we also rename this?

Reply:
(don't need to block merging on another review from me after this fix though)

Reply:
For sure, wasn't sure if that'd be redundant since it's a field within the `Patch` object. Will do, thanks!